components/camel-undertow/src/main/java/org/apache/camel/component/undertow/UndertowEndpoint.java

```diff
@@ -454,14 +458,13 @@ public boolean isPreserveHostHeader() {
     }
 
     public Object getSecurityConfiguration() {
-        return this.securityConfiguration == null ? getComponent().getSecurityConfiguration() : this.securityConfiguration;
+        return this.securityConfiguration;
     }
 
     public void setSecurityConfiguration(Object securityConfiguration) {
         this.securityConfiguration = securityConfiguration;
     }
 
-
     public String getAllowedRoles() {
         return allowedRoles == null ? getComponent().getAllowedRoles() : allowedRoles;
```

Comment:
See prev comment, getter/setter should be plain

Reply:
I'll change it